test/cypress/integration/widgets/blueprints_spec.ts

```diff
@@ -509,6 +512,110 @@ describe('Blueprints widget', () => {
                 cy.contains('button', 'Cancel').click();
             });
         });
+        }
+
+        it('validate form data', () => {
+            openTerraformModal();
+            cy.get('.modal').within(() => {
+                cy.contains('Variables').click().parent().contains('Add').click();
```

Comment:
Probably if you change `clickButton` to be a dual command (see: https://docs.cypress.io/api/cypress-api/custom-commands#Dual-Commands) you can also use it here like:
```suggestion
                cy.contains('Variables').click().parent().clickButton('Add');
```
(other places in this tests using similar construction could be updated as well)

Reply:
Done